src/core/address.h

```diff
@@ -26,6 +26,7 @@
 #define ADDRESS_SERIALIZED_MAX_BYTES (1 + ADDRESS_ED25519_BYTES)
 #define ADDRESS_SERIALIZED_MIN_BYTES (1 + ADDRESS_ALIAS_BYTES)
 
+#define BECH32_ENCODED_ED25519_ADDRESS_STR_LEN 64
 #define BECH32_ENCODED_NFT_ADDRESS 45
 #define BECH32_ENCODED_ALIAS_ADDRESS 45
```

Comment:
remove unused marcos.
```suggestion
#define BECH32_ED25519_ADDRESS_STR_LEN 64
```